.github/uffizzi/k8s/manifests/pg-deploy.yaml

```diff
@@ -19,11 +19,14 @@ spec:
           imagePullPolicy: 'IfNotPresent'
           ports:
             - containerPort: 5432
+          env:
+            - name: PGDATA
+              value: "/data/pgdata"
```

Comment:
I see prettier is complaining about this - it wants single quotes.

Reply:
corrected, thank you for the feedback!